.gitignore

```diff
@@ -53,3 +53,5 @@ dkms.conf
 
 cmake*/
 .vscode/
+
+win32/tests/scratch_ut
```

Comment:
>scratch_ut [](start = 12, length = 10)

Probably shouldn't be checked in

Reply:
Yeah I was just trying out something but didn't want it to end up in the repo. TIL that you can add stuff tp .git/info/exclude to get the same effect, but that file is local so it doesn't need to be pushed.

---
In reply to: [451917869](https://github.com/Azure/azure-c-pal/pull/7#discussion_r451917869) [](ancestors = 451917869)